src/ui/style/Button.js

```diff
@@ -27,49 +27,44 @@ export const StyledButton = styled.button`
     switch (props.buttonType) {
       case "primary":
         return css`
-          background-color: var(--primary-black);
-          color: var(--primary-white);
+          background-color: ${(props) => props.theme.primary};
+          color: ${(props) => props.theme.secondary};
 
           &:disabled {
-            background-color: var(--grey-30);
-            color: var(--grey-5);
+            background-color: ${(props) => props.theme.buttonDisabledBckg};
+            color: ${(props) => props.theme.buttonDisabledBckgText};
```

Comment:
we can change here to `buttonDisabledBG`, what do you think?
just a suggestion, sounds better to my ears :nail_care: